gravitee-reporter-elasticsearch/src/main/resources/freemarker/es7x/mapping/index-template-health.ftl

```diff
@@ -2,6 +2,7 @@
 {
     "index_patterns": ["${indexName}-*"],
     "settings": {
+        <#if indexLifecyclePolicyHealth??>"index.policy.name": "${indexLifecyclePolicyHealth}",</#if>
```

Comment:
According to https://www.elastic.co/guide/en/elasticsearch/reference/7.7/getting-started-index-lifecycle-management.html#ilm-gs-apply-policy

It should be `index.lifecycle.name` instead of `index.policy.name`